test/test_moduleLoad.py

```diff
@@ -38,6 +38,15 @@ def test__dbConfig(self):
 def test_BlockUseSwsssdk():
     # Import swsssdk will throw exception with deprecated message.
     swsssdk_path = os.path.join(modules_path, 'src')
-    result = subprocess.run(["python", "-c", "import swsssdk;exit()"], capture_output=True, cwd=swsssdk_path)
+    result = None
+    python_command = "python"
+    
+    if sys.version_info.major == 3:
```

Comment:
>major [](http://example.com/codeflow?start=24&length=5)

There is a package "python-is-python3", which will symlink python to python3.

You assume python==python2.x, it will not be true if above package exist.

Is it possible to `import swsssdk` in this function `test_BlockUseSwsssdk`?

Reply:
Can't import swsssdk in test case, because if we import it, the code in swsssdk will know it's running inside UT. then will not throw exception.

The version check is to fix the 'python' command does not exist issue. so, if "python-is-python3" installed, this UT still can work.